src/ecr/ecr.cr

```diff
@@ -1,3 +1,22 @@
+# ECR is a template engine for embedding Crystal in HTML.
```

Comment:
Not quite true, it allows embedding it into arbitrary plaintext, for example it's used in the `crystal init` command.